lib/registries/addon/branded/moderation/submissions/controller.ts

```diff
@@ -5,6 +5,8 @@ import { tracked } from '@glimmer/tracking';
 import { RegistrationReviewStates } from 'ember-osf-web/models/registration';
 
 export default class RegistriesModerationSubmissionController extends Controller {
+    queryParams = ['filterState'];
```

Comment:
I'd suggest we make the query param simple `state` and [map it to `filterState` as such](https://guides.emberjs.com/release/routing/query-params/#toc_map-a-controllers-property-to-a-different-query-param-key).
```suggestion
    queryParams = [{
       filterState: 'state',
    }]
```
This is also because we have `view_only` snake-case and `filterState` camelCase. We can simplify `viewMode` as well.